src/main/scala/org/apache/spark/sql/rikai/Image.scala

```diff
@@ -70,5 +70,5 @@ private[spark] class ImageType extends UserDefinedType[Image] {
   */
 @SQLUserDefinedType(udt = classOf[ImageType])
 class Image(val data: Option[Array[Byte]], val uri: Option[String]) {
-  override def toString: String = s"Image(uri='$uri')"
+  override def toString: String = s"Image(uri='${uri.getOrElse("<embedded>")}')"
```

Comment:
should this be "Image(data=<byte array>)" rather than "Image(uri=<embedded>)" for the embedded case?

Reply:
hmm, might be simpler just to do `Image({uri | "<embedded>"})` for now? wdyt